extensions/ql-vscode/src/remote-queries/view/FileCodeSnippet.tsx

```diff
@@ -24,8 +24,8 @@ const getSeverityColor = (severity: ResultSeverity) => {
 const replaceSpaceAndTabChar = (text: string) => text.replaceAll(' ', '\u00a0').replaceAll('\t', '\u00a0\u00a0\u00a0\u00a0');
 
 const Container = styled.div`
-  font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace;
-  font-size: x-small;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
+  font-size: small;
```

Comment:
I see that we still have
```js
const MessageText = styled.div`
  font-size: x-small;
  padding-left: 0.5em;
`;
```
on line 49 below, which means that the message text looks comparatively tiny 🔍 

Could we make the message text the same size as the rest of the container, i.e. also `small`? 😃